src/main/kotlin/uk/gov/justice/hmpps/architecture/defineStyles.kt

```diff
@@ -5,17 +5,49 @@ import com.structurizr.view.Shape
 import com.structurizr.view.Styles
 import uk.gov.justice.hmpps.architecture.annotations.Tags
 
+val softwareColour = "#250469"
+val dbColour = "#1A6926"
+val personColour = "#7E68AB"
+val pipeColour = "#B56107"
+val textColour = "#FFFFFF"
+val providerColour = "#683D0F"
```

Comment:
Probably worth reviewing from colour blindness perspective (not a blocker)
I usually use https://paletton.com for that